readme.md

```diff
@@ -112,20 +138,38 @@ $ npm -v
 * **Question:** What gets logged? Why?
 
 * **Your Answer:**
+PS C:\source\Summer2019\w1-node-ecosystem> node index.js
+Hello, Node!
+Toan Bui
+
 
 * **Question:** What is `module.exports` and what is its _type_ in JavaScript? What is `require` and what is its _type_ in JavaScript?
 
 * **Your Answer:**
 
----
+module.exports
+exported as a module for future use in other js files
+
+ex: in hello.js we have
+module.exports = 'Hello world';
+
+require
+to use modules from other js files
+ex: in index.js we have
+const gretting = require('./hello.js');
+console.log(greeting);
+the output is Hello world
+
+type in JavaScript
+String, Number, Boolean, Undefined, Null, Object, Array, Function
```

Comment:
The question above is asking what the types are of `module.exports` and `require` specifically. `module.exports` is an Object whereas `require()` is a Function.